Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Response/EventData.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Hunted_Mobile.Model.GameModels;
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Hunted_Mobile.Model.Response {
+    public class EventData : ResponseData {
+        public string Message { get; set; }
+        public int TimeLeft { get; set; }
```

Comment:
`TimeLeft` zit toch niet altijd op een `Event`? Ik zou hiervoor een aparte class maken, netzoals je hieronder doet.

Reply:
Jawel, de base class GameEvent in de API heeft `public $timeLeft`.